src/builders.ts

```diff
@@ -413,6 +434,7 @@ export interface ContractCallOptions {
   anchorMode?: AnchorMode;
   postConditionMode?: PostConditionMode;
   postConditions?: PostCondition[];
+  validateWithAbi?: boolean;
```

Comment:
I think there are good use cases for this arg being:
```ts
validateWithAbi?: boolean | ClarityAbi
```

For example
* creating txs in a local dev env where there isn't necessarily a node with the contract ABI available
* caching ABIs
* storing the ABI in your app (for example BNS) for performance improvements

Reply:
I actually had it like this for a while and was on the fence about it. I think you are right and will revert back to this.